Third Party Notices.txt

```diff
@@ -0,0 +1,16 @@
+This file is based on or incorporates material from the projects listed below (Third Party IP). The original copyright notice and the license under which Microsoft received such Third Party IP, are set forth below. Such licenses and notices are provided for informational purposes only. Microsoft licenses the Third Party IP to you under the licensing terms for the Microsoft product. Microsoft reserves all other rights not expressly granted under this agreement, whether by implication, estoppel or otherwise.
+
+---
+
+UnixConsoleEcho
+
+Copyright (c) 2017 Patrick Meinecke
```

Comment:
Shouldn't this be 2018 or at least 2017-2018?  :-)

Reply:
I don't think you need to specify a range, copyright automatically extends quite a few years after initial creation :)

Reply:
Fair enough.  Also, I've been told by our legal department that `Copyright (c)` is redundant.  You only need one of: `Copyright` or `(c)` or `©`.  RE ranges and copyright update, I was told it should be updated whenever the associated source file (or copyrighted artifact) is updated so in our older code-base you see a combination of comma separate years and ranges.  Because of the "copyright extends past the last date for some (large) number of years" we got our legal dept to buy off on us not putting copyright years in our source code.  It got to be too much of a PITA to keep it correct and besides, devs often forget to update copyright notices in source code anyway.  :-)

Reply:
Pretty sure he was just copying my notice :P

If it makes anything easier I can just remove the copyright from the license. Pretty sure it was just a part of whatever template I copied it from :)

Either way I can guarantee I'll never update it :P